### PR TITLE
Greening: Master

### PR DESCRIPTION
**Pull request by IhorBohdan**

### CODE REVIEW

File: cypress/integration/organizations/user-integration-organization.spec.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import TopMenu from '../../support/fragments/topMenu';
+import permissions from '../../support/dictionary/permissions';
+import getRandomPostfix from '../../support/utils/stringTools';
+import TestTypes from '../../support/dictionary/testTypes';
+import users from '../../support/fragments/users/users';
+import newOrganization from '../../support/fragments/organizations/newOrganization';
```
Comment (penfriend) [inline cypress/integration/organizations/user-integration-organization.spec.js:6]: Please import fragment starting from the UpperCase letter

File: cypress/integration/inventory/keyboard-shortcuts.spec.js [hunk 2]
```diff
@@ -67,5 +67,6 @@ describe('ui-inventory: keyboard shortcut', () => {
     InventoryInstanceEdit.addPrecedingTitle(0, precedingTitleValue, isbnValue, issnValue);
     InventoryKeyboardShortcuts.pressHotKey(hotKeys.save);
     InventoryInstance.checkPrecedingTitle(0, precedingTitleValue, isbnValue, issnValue);
+    cy.wait(2000);
```
[the remark this answers is not in the document]
Comment (IhorBohdan) [reply cypress/integration/inventory/keyboard-shortcuts.spec.js:70]: In jenkins the test crashes at the last step, but locally it passes.

File: cypress/integration/organizations/user-integration-organization.spec.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  it('C350762: User can Create and Edit Integrations for an Organization-Vendor (thunderjet)', { tags: [TestTypes.smoke] }, () => {
+    // Found and edit created organization
+    Organizations.searchByParameters('Name', organization.name);
+    Organizations.checkSearchResults(organization);
+    Organizations.chooseOrganizationFromList(organization);
+    // Add first integration and check this
+    Organizations.addIntegration();
+    Organizations.fillIntegrationInformation(integrationName1, integartionDescription1, vendorEDICodeFor1Integration, libraryEDICodeFor1Integration, organization.accounts[0].accountNo, 'Purchase');
+    InteractorsTools.checkCalloutMessage('Integration was saved');
+    Organizations.checkIntegationsAdd(integrationName1, integartionDescription1);
```
[the remark this answers is not in the document]
Comment (IhorBohdan) [reply cypress/integration/organizations/user-integration-organization.spec.js]: Done

File: cypress/integration/organizations/user-integration-organization.spec.js (hunk 1 above)
Comment (IhorBohdan) [reply cypress/integration/organizations/user-integration-organization.spec.js:6]: Done

File: cypress/integration/organizations/user-integration-organization.spec.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    cy.createTempUser([permissions.viewEditCreateOrganization.gui])
+      .then(userProperties => {
+        userId = userProperties.userId;
+        cy.login(userProperties.username, userProperties.password);
+      });
+    Organizations.createOrganizationApi(organization)
+      .then(response => {
+        organization.id = response;
+      });
+    cy.visit(TopMenu.organizationsPath);
```
[the remark this answers is not in the document]
Comment (IhorBohdan) [reply cypress/integration/organizations/user-integration-organization.spec.js:60]: Don't understand.What path?

File: cypress/integration/organizations/user-integration-organization.spec.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
+import TopMenu from '../../support/fragments/topMenu';
+import permissions from '../../support/dictionary/permissions';
+import getRandomPostfix from '../../support/utils/stringTools';
+import TestTypes from '../../support/dictionary/testTypes';
+import users from '../../support/fragments/users/users';
```
[the remark this answers is not in the document]
Comment (IhorBohdan) [reply cypress/integration/organizations/user-integration-organization.spec.js]: Done

**Review by kimkate303 — APPROVED** [review]

**Pull request merged**